pylint/testutils/output_line.py

```diff
@@ -78,19 +83,26 @@ def from_msg(cls, msg):
         )
 
     @classmethod
-    def get_column(cls, column):
+    def get_column(cls, column: Union[int, str]) -> str:
         if not PY38_PLUS:
             return ""  # pragma: no cover
         return str(column)
 
     @classmethod
-    def from_csv(cls, row):
+    def from_csv(
+        cls, row: Union[List[str], Tuple[str, str, str, str, str], str]
+    ) -> "OutputLine":
         try:
-            confidence = row[5] if len(row) == 6 else interfaces.HIGH.name
+            if len(row) == 6:
+                confidence = row[5]  # type: ignore # mypy does not recognize that this cannot fail
```

Comment:
Something we might fix if we do #1590 (By uniformizing all the functional test results)